vars/uploadToS3.groovy

```diff
@@ -1,6 +1,6 @@
 void call(Map args = [:]){
 
-    withAWS(role: "${ARTIFACT_UPLOAD_ROLE_NAME}", roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
+    withAWS(role: 'opensearch-bundle', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
         s3Upload(file: args.sourcePath, bucket: args.bucket, path: args.path)
```

Comment:
@gaiksaya, It would be better we pull the withAWS() role info from credentials
https://github.com/opensearch-project/opensearch-build/blob/main/vars/copyContainer.groovy#L24-L35

Reply:
Its just a name, not the actual value. I checked other places (see PR description) keeping it like that. We have policies in place so random role assumption is restricted.